python/ray/rllib/examples/serving/cartpole_client.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
```

Comment:
we should probably move examples into another repo (or at least something more accessible)